### PR TITLE
Adding pd ID to help on TTx UI

### PR DESCRIPTION
**Pull request by cassiodias**

### CODE REVIEW

**Comment by BrandonMurnane** [comment]
readme and version bump ?

**Comment by LongweiDeng** [comment]
> readme and version bump ?

I probably need to make some changes in this branch. I'll update the ReadMe and Version. 
fyi @cassiodias  @BrandonMurnane

**Comment by cassiodias** [comment]
Please go ahead @longwei, we did that PR yesterday in the last minute :)

**Comment by LongweiDeng** [comment]
Let's not review this PR yet. I'll update it once things are working properly with the TTe story. :) Thx

**Comment by cassiodias** [comment]
I am OK with the PR

File: utils/tte.js [hunk 1]
```diff
@@ -15,22 +15,24 @@ const ZERO = 0;
...
+ * @param {String} id - pager duty id
  * @param {String} team - team paged
  * @param {Array} pdData - list of pagerduty data
  * @returns {Object} - A tte
  */
-function createTTE(team, pdData) {
+function createTTE(id, team, pdData) {
   const tte = {};
-  tte.startTime = pdData.body.log_entries
+  tte.start = pdData.body.log_entries
```
Comment (BrandonMurnane) [inline utils/tte.js]: is the ttx bot using start now or startTime

Comment (cassiodias) [reply utils/tte.js]: Yes

**Pull request merged**